### DISCUSSION TITLE
extending Trigger Mode  brainstorm notes

### DISCUSSION THREAD

**Discussion by listenForward** [bangcorrupt/hacktribe#203] [Ideas]
I really enjoy playing with the Trigger Mode, as manually pushing/gating/re-starting/re-aligning it seems like a fun and different way to really "Play" the device beyond Play/Stop/Mute/Solo/Jump of programmed patters.

I'm exploring how much I can use the e2HackTribe sitting next to my drum kit, so I 
- program separate, often unrelated motifs (arpeggios, break-beats, drones, etc) on various Parts of a given Pattern.
- Set the Trigger Mode to Seq Play
- keep he unit Stopped
- mash the keys (with a drumstick in hand) to fire off little motifs
- release and pull hands back into the drum kit.

However, I find this all hits a wall under the way that the Sequence Play uses the pads only in a "Momentary" way.

Let's brainstorm of some additional Trigger Mode(s) !

I'll go first....inspired  by experiments w/ Eleketon pieces 
- OctaTrack (which can decouple Tracks from the MasterSequencer with momentary/Once/Toggle play  of the tracks by the Onboard Buttons and/or External MIDI Note Events.
- Analog RYTM from Elektron (which runs it's MUTE mode from the Pads).

**1) "Seq OneShot"...** hit a pad to play that Track exactly once, for the full lenght as programmed. Perhaps we could have some variations to quantize Start and Stop to the PadOn and PadOff timing of our fingers...
1.1) _**OneShot(!)**_ to start/Stop immediately, 
1.2)_**OneShot(T)**_ to wait until the next Tempo BPM (Tap event)
1.3) _**OneShot(B)**_ to wait until next Bar
1.4) _**OneShot(P)**_ to wait for full Pattern's MasterSequencer to come back to Step: 1:01

**2) "Seq Toggle" ...** the TRIGGER pad mode acts to Stop and (Re)Start the 16 tracks from play, which makes a Pattern of 16 tracks move toward a primitive Clip Launch-y mode.
Imagine jumping between MUTE mode and Play Toggle TRIGGER ! 

**3) "Seq Choke"...**  Like Seq Toggle, but only allow 1 (or N) Track(s) to Play at Once... so the 16 Tracks can start approximating primitive Clip Launching from the other side.  The Pads are already laid out in 4 little Quadrants / Clumps, so we could have :
_**Seq Choke(4)**_ to split the 16 pads into 4 mute groups, allowing 1 from each Pad Quadrant.

**3) "Seq Note" ...** hit a pad to step/loop through the MIDI Note events sequenced on that Track... could be so the Pads become little "manual playlists".  
Could have variations like :
**_PlayStep(L)_** to sustain each note as long as programmed in the grid, regardless of how you hold/release the pad.
**_PlayStep(H)_** to sustain each note as held/released by the pad.

_**4) "Seq Step" ...**_ hit a pad to step/loop through each 16th note as programmed in the Step Sequencer... including rests. This would be even cooler if the incoming MIDI Note messages (triggered by humans OR automated by sequencers/arpeggios/etc) could turn these Tracks into  "intermediate sub-sequencer modules"

Each of these could get my e2sHackTribe 1 step closer to being the ONLY piece of gear I need by my (electronic) drum-set.

I listen forward reading what other UX others in the Discussion could dream up for the PAD's Trigger Mode(s).

**Comment by javegon** [comment]
Seq Choke Could be nice to star sequences and be stoped by other one. But what do you think about to have choke groups. it is a bit farther.

**Reply by listenForward** [reply]
(sorry for delay).
Good point.
I think we could follow the e2's existing convention ChokeGroups:
since it already has Global note-choke options or tracks 13,14  and for 15/16.

We could have 3 different choke-layout variations for **SeqToggle-_Choke_**

- **"8x2";**  giving us **Pairs**
- > "01/02, 
- > 03/04 
- >...
- >13/14
- >15/16
- **"4x4";** giving us **Squares**
- > 01/02/09/10
- >03/04/11/12
- >05/06/13/14
- >07/08/15/16
- **"2x8";** giving us **Rows**
- > 01/02/.../07/08
- >09/10/.../15/16

**Reply by bangcorrupt** [reply]
I like these as intuitive mute groups.  4 lots of 1x4 could also be good as they sit under the finger tips.

Thank you very much for sponsoring Hacktribe! :smiley:

**Comment by listenForward** [comment]
Interesting. 
If there are Pad Modes for playing Single Hits, First, or SequencePlay, then choke groups could be set up at the "pad sensing" level rather than at the "Voice/Plolyphony management" level, and these could work out independently.
I for one would be fine with choke-groups being global, just like they are for 13/14 and/or 15/16.

**Comment by listenForward** [comment]
Hey BangCorrupt.

You're very welcome.

I tink you're doing incredible work, and hope you continue to chip away at
it.
If you are seeking any user testing/debugging, I have some experience
working remotely w/ the guy/company "Future Artist" in from Germany who
made the his <https://www.youtube.com/watch?v=LyR8zlP5rLk> MIDI Looper tool
<https://www.youtube.com/watch?v=LyR8zlP5rLk>.
Even reviewed his MIDI Looper on my website.
<https://thepatrickrichardson.com/2018/06/13/product-review-for-future-artists-midi-looper-part-1-motivation/>

Before he closed shop, I was helping him for over a year to design and test
the menu/UX for his MIDI Platform prototype
<https://www.youtube.com/watch?v=sEFZlCv7lqc>.

My background includes some experience in  usability research, design
testing, a bit of coding, and a LOT of enthusiasm for gear.
Let me know if/how I might help more beyond my sponsorship.

cheers,
Patrick






On Sat, Mar 16, 2024 at 2:26 PM bangcorrupt ***@***.***>
wrote:

> I like these as intuitive mute groups. 4 lots of 1x4 could also be good as
> they sit under the finger tips.
>
> Thank yo very much for sponsoring Hacktribe! 😃
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bangcorrupt/hacktribe/discussions/203#discussioncomment-8813472>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACX2COASLKDXPATV73F6U3TYYSFDXAVCNFSM6AAAAAAQZB4FOCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQMJTGQ3TE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Patrick Richardson
--
<https://thepatrickrichardson.com/music-lessons/>
Drum Lessons <https://thepatrickrichardson.com/music-lessons/>:
in-person/house-call/remote-video

**Comment by listenForward** [comment]
Hey there HackTribe community.

I've been playing w/ the HackTribe's settings for TriggerMode play, set to SeqPlay and SeqPlay1st...
I appreciate my previous post on this thread are very brain-storm/wish-list material.
...but I want to share/define something of a SMALL suggestion/feature-request that could be a HUGE improvement for me and likely others.

So, for the _stock E2_: the TRIGGER MODE,
to quote Korg's E2 Sampler **Parameter Guide**, section 5, page 6.

> TRIGGER MODE specifies what happens when you strike a trigger pad...
> - **Normal**: Note C4 is sounded

..._(this is fine for drum samples w/ no tuning offset)_
> - **Seq 1st**: The first note recorded in the part is sounded. If not even one note is recorded, the C4 note is sounded

...._this allows you to use the Sequencer to define specific notes/pitches, instead of just the Tune knob... better for one-pad/one-note for pitched parts_
> - **Seq Play**: The sequence recorded in the part plays while you hold down the trigger pad. 

_...**this is what I wish to change**, so that it also had SomeNote/C4 play if the sequence is empty_

If Seq Play could still trigger some sound (once per pad strike) if that Track's Sequence were empty, then we could have TRIGGER MODE in Seq Play and still freely mix/match the 16 pads between manual-trigger/free-play pads VS "run motif" pads, whether the Tracks OSC were set to "Synth", "Sample" or "Audio In(s)",
This is especially clutch for "Audio in" for using pads to gate/dub incoming audio...as I want to still manually gate the audio on that track in Trigger mode, and not HAVE to define specific sequence and run it by Seq Play or Master Transport.

So, the Stock E2 firmware has "if Sequence is empty, trigger a C4 note" for **Seq 1st..**
If the HackTribe OS we can extend that C4-default-trigger over to full **Seq Play**, then the TRIGGER page becomes even more expressive, powerful, and customizable.

What do others think ?
